arbor/include/arbor/cable_cell_param.hpp

```diff
@@ -127,11 +220,18 @@ struct cable_cell_local_parameter_set {
 
 struct cable_cell_parameter_set: public cable_cell_local_parameter_set {
     std::unordered_map<std::string, mechanism_desc> reversal_potential_method;
+    cv_policy discretization = default_cv_policy();
 
     // We'll need something like this until C++17, for sane initialization syntax.
     cable_cell_parameter_set() = default;
-    cable_cell_parameter_set(cable_cell_local_parameter_set p, std::unordered_map<std::string, mechanism_desc> m = {}):
-        cable_cell_local_parameter_set(std::move(p)), reversal_potential_method(std::move(m))
+    cable_cell_parameter_set(
+        cable_cell_local_parameter_set p,
+        std::unordered_map<std::string, mechanism_desc> m = {},
+        cv_policy d = default_cv_policy()
```

Comment:
Can we apply a policy to a cell, then specialise it on a subset of branches?

Reply:
While in principle we could make something like that, it would be quite complex in terms of semantics, given that CVs may cross one or more fork points. I'd recommend that we don't offer such a functionality unless there is a clear need.

As the result of the application of a CV policy is a locset, that set can be amended by user code to cover some special cases. That is, the user code can apply a policty, edit the result, then supply that as an explicit set of CV boundaries.

Reply:
Right.